### DISCUSSION TITLE
discord

### DISCUSSION THREAD

**Discussion by hambuzz45** [Atlas-OS/Atlas#1268] [Help & Support] [answered]
can someone in the owners unban me in the discord pls ?

**Comment by Ast3risk-ops** [comment]
What's your discord username?

**Reply by hambuzz45** [reply]
its buzzblade 45

**Reply by LumiFae** [reply]
@hambuzz45 Discord usernames are only 1 word, no spaces. Also it would be better to get an ID

**Reply by hambuzz45** [reply]
@JayXTQ oh im sorry its buzzblade45 😢

**Reply by LumiFae** [reply]
@hambuzz45 unfortunately we will need your ID it appears, check this resource on how to do so: https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID

**Reply by hambuzz45** [reply]
@JayXTQ 971310549310783508

**Reply by LumiFae** [reply]
@hambuzz45 that user doesn't have any bans on record, are you sure you're banned?

**Reply by hambuzz45** [reply]
@JayXTQ everytime i try to join the server from the link,it always said invalid

**Reply by LumiFae** [reply, marked as answer]
That doesn't mean you are banned, are you using this link? https://discord.atlasos.net/

**Reply by hambuzz45** [reply]
@JayXTQ oh,yeah

**Reply by hambuzz45** [reply]
@JayXTQ i just want to join the server 😭

**Reply by Ast3risk-ops** [reply]
> @JayXTQ i just want to join the server 😭

https://discord.gg/atlasos

**Reply by hambuzz45** [reply]
@Ast3risk-ops why it keep saying that the invite link is invalid or has expired ?

**Reply by H0ggu** [reply]
That link is definitely not invalid or expired, it sounds as if you have been banned on another account as discord bans are IP based too

**Reply by hambuzz45** [reply]
@H0ggu u right,i just told my brother about this,he said that the account that i use got banned in the server bcs he say something on it ,idk ☠️

**Reply by Ast3risk-ops** [reply]
> @hambuzz45 that user doesn't have any bans on record, are you sure you're banned?

Check discord's banlist, forget the bot.

**Comment by Ast3risk-ops** [comment]
And what was that other account @hambuzz45?

**Reply by hambuzz45** [reply]
I only have 1 account

**Reply by Ast3risk-ops** [reply]
> I only have 1 account

Your brother's account that shares your IP, what is it.

**Reply by hambuzz45** [reply]
We use the same account

**Reply by Ast3risk-ops** [reply]
> We use the same account

But there's no ban on it.

**Reply by hambuzz45** [reply]
But my brother told me that he got the account banned in the server 😭
